src/modules/commander/failsafe/failsafe.h

```diff
@@ -132,7 +132,9 @@ class Failsafe : public FailsafeBase
 		JoystickOnly = 1,		// Joystick only
 		RcAndJoystickWithFallback = 2,	// RC And Joystick with fallback
 		RcOrJoystickKeepFirst = 3,	// RC or Joystick keep first
-		StickInputDisabled = 4		// input disabled
+		StickInputDisabled = 4,		// input disabled
+		AscendingPriority = 5,		// ascending source priority
+		DescendingPriority = 6		// descending source priority
```

Comment:
Good catch! In my eyes these redundant enums should not exist. If there's no single point of truth it's just tedious and error-prone to maintain them without them adding value :thinking: